src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,124 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private String isbn;
```

Comment:
The isbn should not be autogenerated, it has a specific meaning and representation strategy, it's "inherent" to the book. The id is a separate value that we use in our database and it's the one that should be autogenerated :)

Reply:
and since it's autogenerated, remember not to include a setter for it 😄